tokio-trace/test_static_max_level_features/main.rs

```diff
@@ -0,0 +1,75 @@
+#[macro_use]
+extern crate log;
```

Comment:
We're testing tokio-trace, not `log`, so we'll want to replace this with 
```suggestion
extern crate tokio_trace;
```